install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
A better place for this file would be `native/install/linux_x86_64.sh`.

Reply:
Done